src/web/theme/modules/ProductList/ProductList.js

```diff
@@ -1,16 +1,54 @@
 import React from "react";
-import ProductItem from "../ProductItem";
-import MediaGrid from "theme/ui/organisms/MediaGrid";
+import ProductHomeView from "./ProductHomeView";
+import compose from "recompose/compose";
+import { withState, withHandlers } from "recompose";
 
-// We are using a specific JSX operator called the "spread" operator that will pass
-// all properties of the given "product" object as props to the ProductItem component
-// See : https://reactjs.org/docs/jsx-in-depth.html#spread-attributes
-const ProductList = ({ products }) => {
+const reassurances = [
+  {
+    accentTitle: "Feel",
+    Title: "the music",
+    contentProductIntro: "Discover our new"
+  },
+  {
+    accentTitle: "Rock",
+    Title: "your body",
+    contentProductIntro: "Fall in love with our classic "
+  }
+];
+
+const ProductList = ({
+  products,
+  currentProductIndex,
+  setCurrentProductIndex
+}) => {
   return (
-    <MediaGrid>
-      {products.map(product => <ProductItem key={product.sku} {...product} />)}
-    </MediaGrid>
+    <div className="product-list">
+      {reassurances.map((reassurance, index) => {
```

Comment:
I don't really like the fact that you have two separate objects representing your product here (reassurance and products[index]).

What you could have done was append the additional data to your products in a withProps HoC, e.g.
```
withProps(props => ({
products: props.products.map((product, index) => ({ ...product, ...reassurances[index] }))
})
```
This way you would only manipulate "products" in the component

Reply:
(Not sure we have the plugin for object spread though in FC Lite :slightly_smiling_face: )

Reply:
OOoooooooooo it's look so cool ! I would like to do it !